webapp/channels/src/components/profile_picture/profile_picture.tsx

```diff
@@ -123,16 +123,16 @@ export default class ProfilePicture extends React.PureComponent<Props> {
                 </span>
                 {this.props.newStatusIcon ? (
                     <StatusIconNew
-                        className={this.props.statusClass}
-                        status={this.props.status}
+                        className={this.props.statusClass || ''}
+                        status={this.props.status || ''}
```

Comment:
Instead of this, you should have changed the props in `status_icon_new.tsx` to be optional. In class components is not necessary because of `defaultProps`, but with functional components we have to be explicit about the optional props.

Reply:
changes revised.